<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWoALauncher</class>
 <widget class="QMainWindow" name="MyWoALauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Winds Of Arithia 2016</string>
  </property>
  <property name="windowIcon">
   <iconset resource="woa_launcher.qrc">
    <normaloff>:/Resources/favicon.png</normaloff>:/Resources/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
image: url(:/Resources/base.png);
}</string>
   </property>
   <widget class="QPushButton" name="button_facebook">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>11</width>
      <height>11</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_facebook
{
border-image: url(:/Resources/facebook.png);
}
QPushButton#button_facebook:pressed
{
border-image: url(:/Resources/facebook.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_play">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>106</width>
      <height>88</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>106</width>
      <height>88</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_play
{
border-image: url(:/Resources/play.png);
}
QPushButton#button_play:pressed
{
border-image: url(:/Resources/play_glow.png);
}
QPushButton#button_play:disabled
{
border-image: url(:/Resources/play_disable.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_twitter">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>80</y>
      <width>12</width>
      <height>11</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_twitter
{
border-image: url(:/Resources/twitter.png);
}
QPushButton#button_twitter:pressed
{
border-image: url(:/Resources/twitter.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>11</width>
      <height>12</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_close
{
border-image: url(:/Resources/close.png);
}
QPushButton#button_close:pressed
{
border-image: url(:/Resources/close.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>528</y>
      <width>178</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar#progress_bar{
border-width: 1px;
text-align: center;
color:rgba(0,0,0,100);
border-radius: 5px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
}

QProgressBar#progress_bar::chunk {
background-color: #6f9aa8;
border-radius: 5px;
}</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>564</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Winds Of Arithia 2016</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ProgressBarOverlay">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>550</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/Resources/Progress1.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>272</y>
      <width>107</width>
      <height>19</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#lineEdit_password
{
	color: rgb(234, 234, 234);
	border-radius: 8px;
	background-color: #314567;
}</string>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>272</y>
      <width>107</width>
      <height>19</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit#lineEdit_username
{
	color: rgb(234, 234, 234);
	border-radius: 8px;
	background-color: #314567;
}</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_youtube">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>80</y>
      <width>12</width>
      <height>11</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_youtube
{
border-image: url(:/Resources/youtube.png);
}
QPushButton#button_youtube:pressed
{
border-image: url(:/Resources/youtube.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_minimize">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>54</y>
      <width>12</width>
      <height>3</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_minimize
{
border-image: url(:/Resources/minimize.png);
}
QPushButton#button_minimize:pressed
{
border-image: url(:/Resources/minimize.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textbrowser_news">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>313</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextBrowser#textbrowser_news
{
	color: rgb(234, 234, 234);
	border-radius: 8px;
	background-color: #314567;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>button_play</zorder>
   <zorder>button_twitter</zorder>
   <zorder>button_close</zorder>
   <zorder>button_facebook</zorder>
   <zorder>ProgressBarOverlay</zorder>
   <zorder>progress_bar</zorder>
   <zorder>label_status</zorder>
   <zorder>lineEdit_password</zorder>
   <zorder>lineEdit_username</zorder>
   <zorder>button_youtube</zorder>
   <zorder>button_minimize</zorder>
   <zorder>textbrowser_news</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
 </tabstops>
 <resources>
  <include location="woa_launcher.qrc"/>
 </resources>
 <connections/>
</ui>
